### DISCUSSION TITLE
Why Fluent UI V9 in not Working in PCF Control

### DISCUSSION THREAD

**Discussion by Nileshkhonde5233** [microsoft/fluentui#33459] [Q&A]
Hi, I am new to PCF (PowerApps Component Framework) controls and learning how to use them. I recently created a basic form in a PCF control project, using Fluent UI V9, which is the latest version. I have created a standard dataset control.

When I added the control to the Dynamics 365 Accounts entity, it worked as expected. My input fields, checkboxes, and radio buttons functioned perfectly.

However, when I tried using the same PCF control in Power Pages, I encountered issues. As shown in the screenshots attached below, Fluent UI is not visible  proper in Power Pages. Additionally, the radio buttons and checkboxes stopped working. Interestingly, when I tried using Fluent UI V8 for the same control, everything worked fine.

Am I missing something, or is Fluent UI V9 not supported in Power Pages yet?

Here is the link of question i raised on community forum
https://community.powerplatform.com/forums/thread/details/?threadid=947299c9-bdb7-ef11-b8e8-6045bddb8ef7

**Comment by ValentinaKozlova** [comment]
Could you please share what errors you encounter? Have you tried some basic controls (like Buttons)? Did it work? Did you use the same code as was provided in examples?
